packages/complex-rubric/src/print.js

```diff
@@ -104,15 +115,25 @@ class ComplexRubricPrint extends HTMLElement {
   get simpleRubric() {
     return this.querySelector(`${RUBRIC_TAG_NAME}#simpleRubric`);
   }
+  get rubricless() {
+    return this.querySelector(`${RUBRIC_TAG_NAME}#rubricless`);
+  }
 
   connectedCallback() {
     this._render();
   }
 
   _render() {
-    this.innerHTML = this._type === RUBRIC_TYPES.SIMPLE_RUBRIC
-      ? `<${RUBRIC_TAG_NAME} id="simpleRubric" />`
-      : `<${MULTI_TRAIT_RUBRIC_TAG_NAME} id="multiTraitRubric" />`;
+    let rubricTag;
+    if (this._type === RUBRIC_TYPES.SIMPLE_RUBRIC) {
+      rubricTag = `<${RUBRIC_TAG_NAME} id="simpleRubric" />`;
+    } else if (this._type === RUBRIC_TYPES.RUBRICLESS) {
+      rubricTag = `<${RUBRIC_TAG_NAME} id="rubricless" />`;
+    } else {
+      rubricTag = `<${MULTI_TRAIT_RUBRIC_TAG_NAME} id="multiTraitRubric" />`;
+    }
+
+    this.innerHTML = rubricTag;
```

Comment:
maybe we can make this a bit prettier and use an object and render based on that
something like this: 
 ```
const rubricTags = {
        [RUBRIC_TYPES.SIMPLE_RUBRIC]: `<${RUBRIC_TAG_NAME} id="simpleRubric" />`,
        [RUBRIC_TYPES.RUBRICLESS]: `<${RUBRIC_TAG_NAME} id="rubricless" />`,
        [RUBRIC_TYPES.MULTI_TRAIT_RUBRIC]: `<${MULTI_TRAIT_RUBRIC_TAG_NAME} id="multiTraitRubric" />`,
    };
```
and then just render like this
```
const rubricTag = rubricTags[this._type] || '';

this.innerHTML = rubricTag;
```

It's just a suggestion to make this more easy to read